src/components/molecules/ChatPoll/ChatPoll.tsx

```diff
@@ -65,9 +68,9 @@ export const ChatPoll: React.FC<ChatPollProps> = ({
     "ChatPoll--me": isMine,
   });
 
-  const handleVote = useCallback(
-    (question) =>
-      voteInPoll({
+  const [{ loading: isVoting }, handleVote] = useAsyncFn(
+    async (question) =>
+      await voteInPoll({
         questionId: question.id,
         pollId: id,
       }),
```

Comment:
You don't need await here

Reply:
fixed in 1d3b5688b0f98e34f710a6d2cb8fb2ca296afcb7